File: challenges/2-idade-em-dias/ruby/jailsonsf/desafio-2.rb
days = ARGV.map(&:to_i)[0]

years = (days / 365).floor
days -= years * 365
months = (days / 30).floor
days -= months * 30

puts "#{years} ano(s)"
puts "#{months} mes(s)"
puts "#{days} dia(s)"

File: challenges/2-idade-em-dias/javascript/JoaoSortudo/desafio2.js
let n = process.argv[2]

    let ano = parseInt(n / 365);
    let mês = parseInt((n % 365) / 30);
    let dias = (n % 365) % 30;
    console.log(ano,"ano(s)", mês, "mes(es)", dias, "dia(s)");
